data/collections.json

```diff
@@ -281,8 +281,8 @@
       "documents": "9"
     },
     "industries": {
-      "en": "Subset of services used by Open Terms Archive",
-      "fr": "Sous-ensemble de services utilisés par Open Terms Archive"
+      "en": "Services used by Open Terms Archive",
+      "fr": "Services utilisés par Open Terms Archive"
```

Comment:
It's an approximation that I'm comfortable with, but we thought with @Ndpnt that it was better to be precise and say that it's a sub-set of services, because we don't track all the services used by the OTA team, and there are services that aren't necessary for the engine.